packages/xdl/src/detach/IosIPABuilder.js

```diff
@@ -124,8 +125,8 @@ export default function createIPABuilder(buildParams) {
       }
     );
     const plistRaw = output.join('');
-    const plistData = _.attempt(plist.parse, plistRaw);
-    if (_.isError(plistData)) {
+    const plistData = attempt(plist.parse, plistRaw);
```

Comment:
```suggestion
    const plistData = typeof plist.parse === 'function' ? plist.parse(plistRaw) : undefined;
```
Something like this?

Reply:
Went with 
```ts
    try {
      return plist.parse(plistRaw);
    } catch (error) {
      throw new Error(`Error when parsing plist: ${error.message}`);
    }
```
`attempt` followed by `isError` must be the weirdest way to catch errors I've seen this week 😄